packages/modules/fsm/src/pages/employee/Inbox.js

```diff
@@ -21,6 +21,13 @@ const Inbox = ({ parentRoute, isSearch = false, isInbox = false }) => {
   const [pageSize, setPageSize] = useState(10);
   const [sortParams, setSortParams] = useState([{ id: "createdTime", desc: false }]);
   const [searchParams, setSearchParams] = useState(() => {
+    // console.log("hey find filters here",Digit.SessionStorage.get("digit.fsm.inbox.filter"), isSearch)
+    if (Digit.SessionStorage.get("digit.fsm.inbox.filter") && !isSearch) {
```

Comment:
@mahendra-egov has already done a PR for this change, @saurabh-egov have you looked at it?